File: C/section06/S06E09.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    float pollutionIndex;

    printf("Please enter the air pollution index value: \n");
    scanf("%f", &pollutionIndex);

    if (pollutionIndex >= 0.5) {
        printf("All industries must suspend their activities.");
    } else if (pollutionIndex >= 0.4) {
        printf("The industries of the first and second industry groups must suspend their activities.");
    } else if (pollutionIndex >= 0.3) {
        printf("Only the industries of the first industry group must suspend their activities.");
    } else {
        printf("The air pollution level is at an acceptable index.");
    }

    return 0;
}

File: Python/section06/S06e09.py
pollutionIndex = float(input ("Please enter the pollution index value: "))

if pollutionIndex >= 0.5:
    print("All industries groups must suspend their activities.")
elif pollutionIndex >= 0.4:
    print("Industries of groups 1 and 2 must suspend their activities.")
elif pollutionIndex >= 0.3:
    print("Industries of group 1 must suspend their activities.")
else:
    print("The pollution index is at an acceptable level.")